### ISSUE TITLE
Feature to take an exported spreadsheet of Twitter followers 

### ISSUE THREAD

**Issue by jaredgoodman03** [NicolasConstant/BirdsiteLive#73]
I know there's [websites](https://simpleample.com/how-export-twitter-followers-excel-file-free/) that let you export Twitter followers to a spreadsheet. It would be really cool to be able to upload this spreadsheet to BerdsiteLive, and it gives you a CSV that you can import into Mastodon. This would help a lot of people migrate from Twitter to the fediverse.

I don't know C#, but I could host my own service that does this with Python or something if you don't feel like implementing it directly into this repo.

**Comment by NicolasConstant** [comment]
I understand the need, and it would be indeed useful and an easy way to import all Twitter followers. 
But this would also encourage people to import hundreds of followers with a single click, and unless you're hosting your own instance, it wouldn't be a good thing to promote: 

This project rely on Twitter's API, and it has a limitation on how much calls you can perform per 15 mins and per day, therefore we should encourage people to choose wisely the accounts they really want to follow instead of blindly importing all accounts, that would saturate the instance very fast and will degrade its performance (instead of having updates each 15 mins it could decrease to 1 per hour, one every 2 hours, etc). 

That said, I can totally add this kind of functionality, but I would make it disabled by default (and on the official instance) and it would only be available to people that would self-host their own instance (if someone want to follow hundreds of users on their own personal server, that's ok and that would work).

**Comment by jaredgoodman03** [comment]
Yeah, that makes sense.
I know Nitter has a feature to get RSS feeds of someone's page, just by going to `nitter.net/usernamehere/rss`. Using that instead could allow this project to not be limited by Twitter's API calls.

**Comment by NicolasConstant** [comment]
Yes I'm aware of the scrapping capability of Nitter - using the "unofficial twitter API" - but that's something that would be less convenient to deploy (you have some more dependencies) and scrapped content can have some issues too. 

There is some mirror bots using Nitter on the Fediverse too, and their content was always way less interesting (images instead of videos, etc) than the ones I've made with Twitter API. 

It's a tradeoff, clearly. 

I may receive a PR one day to support the scrapping strategy instead of the API (that would be a nice option to provide) and I will be happy to merge it obviously. 

Also, the fact that this project isn't scalable as it is now isn't something that bother me, on the contrary: it will motivate people to open their own instances, and therefore decentralize. Which is good, even on a simple privacy aspect.
